sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/encoders/ProductEncoder.scala

```diff
@@ -61,7 +62,31 @@ object ProductEncoder {
 
   def extractorFor(
```

Comment:
Didn't these methods used to have scaladoc?  Can you put that back when you undo the split.